packages/kit/src/exports/vite/index.js

```diff
@@ -886,7 +886,13 @@ async function kit({ svelte_config }) {
 					await build_service_worker(
 						out,
 						kit,
-						vite_config,
+						{
+							...vite_config,
+							build: {
+								...vite_config.build,
+								minify: initial_config.build?.minify ?? 'esbuild'
```

Comment:
any reason to use initial_config instead of vite_config here, and/or not fall back to its minify option first?

Reply:
For some reason, `vite_config` gets changed to the Vite client build defaults instead of the user's config. For that same reason we're using initial_config below: https://github.com/sveltejs/kit/blob/33600ee3a7aaf5d93b808b525c773b3b8580bd51/packages/kit/src/exports/vite/index.js#L808